server.go

```diff
@@ -296,6 +368,7 @@ func (s *Server) HandleTracePacket(packet []byte) {
 	// Unlike metrics, protobuf shouldn't have an issue with 0-length packets
 	if len(packet) == 0 {
 		log.Error("received zero-length trace packet")
+		return
```

Comment:
The comment above this `if` suggests that maybe this was intentional? If not, maybe the comment could just be dropped if you are going to early return on empty data, since I'm not sure what the point of the comment is?